app/src/main/java/com/example/tanya/godeltechchallengeandroid/domain/entity/Word.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.tanya.godeltechchallengeandroid.domain.entity
+
+data class Word(val word: String,
+                val count: Int,
+                val isStillComputing: Boolean = false)
```

Comment:
Please double check if we are still using the `isStillComputing` property. How I remember we changed a presentation logic and it is not used anymore.